support-cycle.md

```diff
@@ -0,0 +1,85 @@
+
+##### Validating Chrome support
+
+We validate our support for Chrome by running Continuous Integration testing for the latest version of Chrome. Older versions aren't tested because this is not easily available in our Continuous Integration environments.
+
+#### Safari (MacOS and iOS)
+
+Safari is tied to the MacOS and iOS operating system releases, of which Apple supports the last 3 releases in each (this equates to roughly 3 years, as a new MacOS and iOS release is made every year), however Apple also releases updates for Safari versions for the last 3 _desktop_ operating system versions. [Apple's support notes for security releases can be found here](https://support.apple.com/en-us/HT201222). Information around Safari releases can be a little tricky to find, and Apple does not have sufficient support documentation around this - the best place to find this information is on the [Wikipedia entry for Safari version history](https://en.wikipedia.org/wiki/Safari_version_history). To demonstrate how the release cycle works; iOS 8 and MacOS 10.10 both shipped with Safari 8, iOS9 and MacOS 10.11 shipped with Safari 9, iOS10 and MacOS 10.12 shipped Safari 10 and iOS11 and MacOS 10.13 shipped with Safari 11. Safari 10 was backported as an update to MacOS 10.11 and 10.10 users, as was Safari 11 to MacOS 10.12 and 10.11 users. The iOS version of Safari remains fixed.
+
+##### Validating Safari support
+
+We validate Safari support by running Continuous Integration testing for the oldest supported version of iOS and MacOS, as well as the latest supported versions of both. Any versions inbetween are not tested as it is unlikely that there will be failures in specific versions between the oldest supported version and the latest release that aren't reflected in one or the other.
+
+#### Edge
+
+Microsoft have no documentation of the Microsoft Edge Support Cycle. We currently support the latest release of Microsoft Edge, with soft-support for public up to a year old (that is, we do not test older releases, but if a user finds a bug we will fix it).
+
+##### Validating Edge support
+
+We validate support of Edge by running Continuous Integration testing for the latest version. Older versions aren't tested due to build time constraints.
+
+#### Internet Explorer
+
+[Microsoft have officially stopped support for Internet Explorer 10 and below since January 2016](https://www.microsoft.com/en-gb/windowsforbusiness/end-of-ie-support), but still continue to support Internet Explorer 11 in Windows 7, 8 and 10. The end-of-life date for Windows 10 is October 25 as explained in the [Microsoft LifeCycle Fact sheet](https://support.microsoft.com/en-us/help/13853/windows-lifecycle-fact-sheet), and so we can expect to support Internet Explorer 11 until then. We do not support older versions of Internet Explorer.
```

Comment:
Maybe this is just me (and I'm not a native english speaker), but when I read `October 25` I think of the 25th day of October. Maybe this could be reword to `October of 2025`?